packages/client/katello-host-tools/katello-host-tools.spec

```diff
@@ -214,9 +232,6 @@ rm %{buildroot}%{plugins_dir}/__init__.py
 
 # executables
 mkdir -p %{buildroot}%{_sbindir}
-%if %{yum_install} || %{zypper_install}
-cp bin/* %{buildroot}%{_sbindir}/
-%endif
 %if %{dnf_install}
 cp extra/katello-tracer-upload-dnf %{buildroot}%{_sbindir}/katello-tracer-upload
 %endif
```

Comment:
possibly?  But we don't have the ability to truly test it, and I'm hesitant to make such a change in this attempt to keep things as 1-1 as possible to ensure a successful update to 4.2.3.

Leaving the sed commands below in should not affect those files in any way if they're already in the desired final state.

Reply:
@sbernhard could you have a look at this? I recall ATIX does deliver on SuSE. And ATIX doesn't, then I'd consider dropping all the SuSE bits. Because unused code is likely broken.

Reply:
Yes, we are still building SLES clients.

Reply:
We discussed this topic internally and came to the conclusion to deliver the necessary SLES packages similar as we already do it with Debian/Ubuntu packages at https://apt.atix.de

Reply:
Please, don't drop it. We are using this spec files and will then, based on this spec, deliver rpm packages for SLES

Reply:
Ok, because that was what I was after. Can you check if this bit is still needed with this PR:
https://github.com/theforeman/foreman-packaging/blob/166bb012380ce7b7791fef33b0e86e03aaa4b055/packages/client/katello-host-tools/katello-host-tools.spec#L224-L231

I think it's redundant and can be dropped, but have no way to verify this.

Reply:
For timing's sake, can we handle this off of this PR?  Depending on result I can file a new PR with the fixes in them after this one is merged.

Reply:
I don't consider it blocking, but something I'd like to take care of. If we can simplify the already too complex spec, then I'd be all for it. We can also clean up the Fedora bits but I won't hold back on that either.

Reply:
If python3 on a SLES should be used, it need to be "python3" instead of python.

Reply:
In version 4 the files are using generated wrappers and I'd expect the shebang to match the Python version used in the build. So can you please perform a build without those lines and see which shebang ends up being used? I really think it should be redundant now if SLES has their build systems set up correctly.

I took a quick look at it, but the builds fail: https://build.opensuse.org/package/show/home:ekohl/katello-host-tools has:
```
[   20s] found conflict of katello-host-tools-4.2.3-150400.5.1.noarch with katello-host-tools-tracer-4.2.3-150400.5.1.noarch:
[   20s]   - /usr/lib/python3.6/site-packages/katello/tracer.py
[   20s]   - /usr/lib/python3.6/site-packages/katello/zypper_tracer.py
```

This made me look at the EL9 builds and there we also see `/usr/lib/python3.9/site-packages/katello/tracer.py` show up in both `katello-host-tools` and `katello-host-tools-tracer`. Given `katello-host-tools-tracer-4.2.3-1.el9.noarch.rpm` requires `katello-host-tools`, I think there it would also lead to uninstallable packages, but DNF appears to deal with it. Maybe because they're the same file? Does lead to an interesting situation:
```console
# rpm -qf /usr/lib/python3.9/site-packages/katello/tracer.py
katello-host-tools-4.2.3-1.el9.noarch
katello-host-tools-tracer-4.2.3-1.el9.noarch
```

I'd still prefer to keep `tracer.py` out of `katello-host-tools`.

I've submitted https://github.com/Katello/katello-host-tools/pull/142 which moves tracer into its own directory, which should make it easier in the future.

Reply:
tracer bits should now be excluded.

i've removed the sles sed.